NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -86,6 +86,65 @@ internal bool Save(PersistedWorldData persistedData, string saveDir)
             }
         }
 
+        public bool BackUp(string saveDir)
+        {
+            string backupDir = Path.Combine(saveDir, "Backups");
+            if (!Directory.Exists(backupDir))
+            {
+                Directory.CreateDirectory(backupDir);
```

Comment:
Directory.CreateDirectory already check for path before creating it

https://learn.microsoft.com/fr-fr/dotnet/api/system.io.directory.createdirectory?view=netframework-4.7.2

Reply:
The CreateDirectory should be performed in the try catch also, since it can throw an exception.